activerecord/lib/active_record/scoping/named.rb

```diff
@@ -19,7 +19,7 @@ module ClassMethods
         #
         # You can define a scope that applies to all finders using
         # {default_scope}[rdoc-ref:Scoping::Default::ClassMethods#default_scope].
-        def all
+        def all(all_queries: nil)
```

Comment:
Do we want to hide this argument in the docs?

Reply:
I think this is fine.